src/components/__tests__/snapshot.test.js

```diff
@@ -0,0 +1,89 @@
+import TestRenderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import {
+  BrowserRouter,
+  BrowserRouter as Router,
+} from 'react-router-dom';
+import Details from '../Details';
+import Home from '../Home';
+import Cocktail from '../Cocktail';
+import Listhome from '../Listhome';
+import Listdetails from '../Listdetails';
+import Navbar from '../Navbar';
+import store from '../../redux/configureStore';
+
+describe('Render Navbar', () => {
+  it('Renders the navbar', () => {
+    const header = TestRenderer.create(
+      <Provider store={store}>
+        <Router>
+          <Navbar />
+        </Router>
+      </Provider>,
+    ).toJSON();
+    expect(header).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Well done with testing the rendering of the components so far.👏 However, you still have to write some unit and integration tests for your API functions and the Redux store by mocking. 😄 🙏 
_Please take a look at the requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests%20for%20pure%20functions%20in%20your%20Redux%20code.)._

Reply:
add state filter for name in home also add stateless for language in cocktails page